examples/nextjs-spa/components/sdk.tsx

```diff
@@ -0,0 +1,11 @@
+import { Configuration, Identity, V0alpha2Api } from "@ory/client"
+import { edgeConfig } from "@ory/integrations/next"
+
+// Create a new Ory API client
```

Comment:
```suggestion
// Create a new Ory API client
// This will default to the Ory playground project if no environment variable is set. 
// Set the `ORY_SDK_URL` to your Ory CLI tunnel e.g. http://localhost:4000
```